### PR TITLE
Schema changes - Transition filing

### PR DESCRIPTION
**Pull request by lekshmimallika-aot**
*Issue #:* /bcgov/entity#5182

*Description of changes:*
Added the schema and sample data for a transition filing
Added tests

All the business specific information like identifier, legal type and legal name will be present in the business section of the filing outside the transition object. As we are not altering any of the business specific info, it need not be present inside the transition object

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-schemas license (Apache 2.0).

### CODE REVIEW

**Review by kialj876 — APPROVED** [review]
Nice work!

File: src/registry_schemas/schemas/transition.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    "parties": {
+      "type": "array",
+      "items": {
+        "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/parties#/definitions/party"
+      }
+    },
+    "shareStructure": {
+      "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/share_structure"
+    },
+    "hasProvisions": {
```
Comment (thorwolpert) [inline src/registry_schemas/schemas/transition.json:35]: Is this different than the resolutionDates or hasRightsOrRestrictions already in share structure?

**Review by thorwolpert — COMMENTED** [review]
Just 1 question on provisions.

File: src/registry_schemas/schemas/transition.json (hunk 1 above)
Comment (lekshmimallika-aot) [reply src/registry_schemas/schemas/transition.json:35]: Yes, this is the pre existing company provisions that we store in the businesses table (restriction_ind column)

Comment (thorwolpert) [reply src/registry_schemas/schemas/transition.json:35]: right.
thanks for walking through in chat.

**Review by thorwolpert — APPROVED** [review]

**Comment by thorwolpert** [comment]
thanks!!

**Pull request merged**